### PR TITLE
[WIP]Renew website

### PR DESCRIPTION
**Pull request by Aissaoui-Ahmed**
Switch Wordpress to Github pages 

WIP pages :

- [ ] achievements
- [ ] e-marketing
- [ ] training
- [ ] our-dna
- [ ] contact-us
- [ ] events
- [ ] solutions/web
- [ ] solution/mobile
- [ ] solutions/desktop
- [ ] 404
- [ ] open-source
- [ ] blog 
- [ ] ...

### CODE REVIEW

**Review by KassemHachi — APPROVED** [review]

File: _data/nav.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+- link: /achievements
+  title: achievements
+- all_item:
+    - link: /solution/web
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
    - link: /solutions/web
```

File: _data/nav.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      title: web
+    - link: /solution/mobile
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
    - link: /solutions/mobile
```

File: _data/nav.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      title: Mobile
+    - link: /solution/desktop/
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
    - link: /solutions/desktop/
```

File: _data/nav.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      title: Desktop
+  link: /Solution/
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
  link: /Solutions/
```

File: _data/nav.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  title: Solution
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
  title: Solutions
```

File: _data/nav.yml [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  sub_items: "yes"
+- link: /e-marketing/
+  title: e-Marketing
+- link: /formations/
+  title: Formations
+- link: /about
+  title: "Who are we?"
```
Comment (Boukhtam) [inline _data/nav.yml]: I'm confused here, between English and French
```suggestion
  title: "Our DNA"
```

File: _data/nav.yml [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    - link: /solution/desktop/
+      title: Desktop
+  link: /Solution/
+  title: Solution
+  sub_items: "yes"
+- link: /e-marketing/
+  title: e-Marketing
+- link: /formations/
+  title: Formations
+- link: /about
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
- link: /our-dna
```

File: _data/nav.yml [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  title: "Who are we?"
+- link: /contact-us/
+  title: Contact
+- link: /events
```
Comment (Boukhtam) [inline _data/nav.yml:22]:
```suggestion
- link: /events
  title: Events
```

File: _data/nav.yml [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    - link: /solution/mobile
+      title: Mobile
+    - link: /solution/desktop/
+      title: Desktop
+  link: /Solution/
+  title: Solution
+  sub_items: "yes"
+- link: /e-marketing/
+  title: e-Marketing
+- link: /formations/
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
- link: /training/
```

File: _data/nav.yml [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+  title: Formations
```
Comment (Boukhtam) [inline _data/nav.yml]:
```suggestion
  title: Training
```

**Review by Boukhtam — COMMENTED** [review]
Changes you made are too large, so I can't keep track of it.

**Review by celyes — COMMENTED** [review]
post screenshots for the main page

**Comment by Aissaoui-Ahmed** [comment]
> post screenshots for the main page

@celyes  see live preview [SoftVenue](https://aissaoui-ahmed.github.io/softvenue/)

**Review by celyes — COMMENTED** [review]
i suggest you delete `our-dna` section at all... we are not a company yet...

**Review by Salem-Abderaouf — APPROVED** [review]

**Pull request closed without merging**